index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="Product Landing Page">
+    <title>Product Landing Page for Credit and Saving company</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body id="body">
+ <div id="page-wrapper">
+   <header id="header"> 
```

Comment:
Please delete the div that wraps your header. So that you know, the header tag is enough for this need.

Reply:
- [Optional] Great job for adding the comment politely and with reasons 👍,  I think it would be better if you use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.